ipatests/test_ipaserver/test_install/test_installer.py

```diff
@@ -0,0 +1,185 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors.  See COPYING for license
+#
+from __future__ import absolute_import
+
+import six
+
+if six.PY2:
+    from abc import ABCMeta
+
+    class ABC(object):
+        __metaclass__ = ABCMeta
+else:
+    from abc import ABC  # pylint: disable=no-name-in-module
+
+from abc import abstractmethod
+from collections import namedtuple
+import itertools
+import tempfile
+
+from ipatests.util import assert_equal
+from ipaserver.install.ipa_replica_install import ReplicaInstall
+
+Keyval = namedtuple('Keyval', ['option', 'value'])
+
+
+class InstallerTestBase(ABC):
```

Comment:
Please use six like in other places: ``class InstallerTestBase(six.with_metaclass(abc.ABCMeta, object)):``. This will also get rid of the PEP 8 violation ``E402 module level import not at top of file``